GoogleMapsApi/Entities/Geocoding/Request/GeocodingComponents.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+namespace GoogleMapsApi.Entities.Geocoding.Request
+{
+    public class GeocodingComponents : Dictionary<string, string>
```

Comment:
Please do. `GeocodingComponents` is NOT a Dictionary.

Generally, you should prefer composition over inheritance.